theme/islandora-manuscript-manuscript.tpl.php

```diff
@@ -9,4 +9,19 @@
     <?php print $viewer; ?>
   </div>
 <?php endif; ?>
-<!-- @todo Add table of metadata values -->
+<?php if ($display_metadata): ?>
+    <div class="islandora-collection-metadata">
```

Comment:
This class should be unique to the manuscript module so any existing selectors don't collide, something like `islandora-manuscript-metadata`?